Tools/scripts/build_binaries.sh

```diff
@@ -267,6 +267,26 @@ build_arducopter() {
         copyit ArduCopter-v2.px4 $ddir $tag &&
         test ! -f ArduCopter-v4.px4 || copyit ArduCopter-v4.px4 $ddir $tag
     done
+    for b in bebop ; do
+        checkout ArduCopter $tag $b || {
+            echo "Failed checkout of ArduCopter $b $tag"
+            error_count=$((error_count+1))
+            continue
+        }
+        skip_board $b && continue
+        echo "Building ArduCopter $b binaries"
+        ddir=$binaries/Copter/$hdate/$b
+        skip_build $tag $ddir && continue
+        make clean || continue
+        make $b -j4 || {
```

Comment:
@rmackay9 Is that urgent? I'm thinking about reworking the *build_binaries.sh* file but I don't have an exact timeline (max two weeks I hope).